src/playlist-loader.js

```diff
@@ -496,6 +504,18 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
 
       loader.state = 'HAVE_MASTER';
 
+      if (req.responseURL) {
+        // checking, whether manifest request ended with redirect
+        if (srcUrl !== req.responseURL) {
+          srcUrl = req.responseURL;
+        }
+      } else if (!isXhrRedirectSupportWarned) {
+        isXhrRedirectSupportWarned = true;
+        log.warn(
+          'Current browser does not support redirects for playlist requests.'
```

Comment:
I'm not sure if we have to warn here, since most playlists will not need a redirect (meaning browsers that don't support redirects will warn even if there's nothing to act on).

Reply:
did you mean this comment?

>// if the playlist returns a 302 redirect for manifest, ...

or which comment from above?